tests/performance_tests/performance_utils.py

```diff
@@ -17,14 +17,14 @@
 
 def write_summary_spec(file, keywords):
     content = [
-        ("INTEHEAD", [1, 100]),
+        ("INTEHEAD", array([1, 100], dtype=numpy.int32)),
```

Comment:
btw, you can also do `array(...).astype(np.int32)`, which applies for all the others.

Reply:
are you sure that will not create the array with double values and then do the astype-conversion?

Reply:
Good point!